internal/pkg/service/cli/cmd/cmd.go

```diff
@@ -203,8 +207,8 @@ func NewRootCommand(stdin io.Reader, stdout io.Writer, stderr io.Writer, envs *e
 		StatusCommand(p),
 		sync.Commands(p),
 		ci.Commands(p),
-		local.Commands(p, envs),
-		remote.Commands(p, envs),
+		local.Commands(p, osEnvs),
+		remote.Commands(p, osEnvs),
```

Comment:
envs argument is unused in both functions, please remove it.

Reply:
it solved https://github.com/keboola/keboola-as-code/pull/1624/commits/9933e67cc02a094d48ff5acf77815746d3fd83a4